### PR TITLE
Improve URLs for Chart and Dashboard ModelViews

### PR DESCRIPTION
**Pull request by mistercrunch**
Prior to this, the ModelView for Chart and Dashboard would be
at `/slicemodelview/list/` and `/dashboardmodelview/list/`.

Now we have cleaner URLs at `/chart/list/` and `/dashboard/list/`

@betodealmeida @graceguo-supercat 

Note that this may break some bookmarks, but I feel like it's worth doing.

### CODE REVIEW

File: superset/connectors/sqla/models.py [hunk 1]
```diff
@@ -751,6 +751,7 @@ def query(self, query_obj):
         qry_start_dttm = datetime.now()
         sql = self.get_query_str(query_obj)
         status = QueryStatus.SUCCESS
+        print(sql)
```
Comment (betodealmeida) [inline superset/connectors/sqla/models.py]: `dd`

File: superset/viz.py [hunk 2]
```diff
@@ -371,6 +371,7 @@ def get_df_payload(self, query_obj=None):
                     self.query = cache_value['query']
                     self._any_cached_dttm = cache_value['dttm']
                     self._any_cache_key = cache_key
+                    print(cache_value)
```
Comment (betodealmeida) [inline superset/viz.py]: `dd`

**Review by betodealmeida — APPROVED** [review]
I agree it's much better and worth breaking bookmarks.

Note that you forgot two `print` calls in the PR.

**Comment by mistercrunch** [comment]
We might want to do this on more `ModelViews`

**Pull request merged**